src/ZTUtils/Zope.py

```diff
@@ -297,7 +303,11 @@ def complex_marshal(pairs):
 def simple_marshal(v):
     if isinstance(v, str):
         return ''
+    if isinstance(v, binary_type):
+        # Py 3 only
+        return ':bytes'
```

Comment:
Does that need an explicit `if PY3` check?  Under Python 2, that will be *any* `str` instance.